src/test/integration/integration-test.py

```diff
@@ -96,8 +87,12 @@ def test_get_non_authority_onids(self, endpoint='/spendingauthority'):
     def test_bad_request_response(self, endpoint='/spendingauthority'):
         bad_params = [{'onid': ''}, {}]
         for bad_param in bad_params:
-            params = bad_param
-            response = utils.make_request(self, endpoint, 400, params=params)
+            response = utils.make_request(
+                self,
+                endpoint,
+                400,
+                params=bad_param
+                )
```

Comment:
Styling comment: since you only have one char over the max line length limit, I'd rather rename `bad_param` to be `param` or something else shorter, so that you should be able to still keep all of them in the same line.